.github/workflows/test.yaml

```diff
@@ -2,6 +2,8 @@ name: test
 permissions:
   contents: read
 on: [push, pull_request]
+
+# Copied from https://github.com/rustls/rustls/blob/ba49f6f0b5704ecc0c7a15e5ab8819bf22edc4cb/.github/workflows/build.yml
```

Comment:
I'm guessing this comment is left over from the CI changes that were punted out of this PR?